src/components/m-datepicker/index.js

```diff
@@ -87,7 +84,7 @@ class AXADatepicker extends BaseComponentGlobal {
   // }
   didRenderCallback() {
     // TODO see if it has to be today and if so pass here value
-    this.datepicker.init(getAttribute(this, 'output-iso'));
+    // this.datepicker.init(getAttribute(this, 'output-iso'));
```

Comment:
This looks broken, seems that initialisation is never called now 🤔

Reply:
I removed all datepicker changes. they were not meant to be in this PR.